RELEASE.rst

```diff
@@ -0,0 +1,16 @@
+RELEASE_TYPE: minor
+
+This release improves validation of numeric bounds for some strategies.
```

Comment:
I think you're not just improving validation, but *fixing a bug*, and it's worth calling that out. The old code gave examples outside the bounds, contrary to what we said it would do.

The current version of this PR leaves that one case where we still do this and give a deprecation warning, but even if you don't make that an error (as I suggest below), you're still fixing the general case.

Reply:
Made into an error and noted below.